app/views/layouts/base.html.erb

```diff
@@ -33,7 +33,6 @@
     <%= javascript_include_tag *webpack_asset_paths('vendor', :extension => 'js'), "data-turbolinks-track" => true %>
     <%= javascript_include_tag *webpack_asset_paths('bundle', :extension => 'js'), "data-turbolinks-track" => true %>
     <%= javascript_include_tag 'application', "data-turbolinks-track" => true %>
-    <%= webpacked_plugins_with_global_js %>
```

Comment:
what does this do and why do we need it?

Reply:
probably deleted during rebased...